processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByMergingQueryRunnerV2.java

```diff
@@ -169,11 +169,11 @@ public GroupByMergingQueryRunnerV2(
                 // This will potentially block if there are no merge buffers left in the pool.
                 final long timeout = timeoutAt - System.currentTimeMillis();
                 if (timeout <= 0 || (mergeBufferHolder = mergeBufferPool.take(timeout)) == null) {
-                  throw new QueryInterruptedException(new TimeoutException());
+                  throw new TimeoutException();
```

Comment:
BlockingPool.take() now throws RuntimeException instead of InterruptedException, and the below ```catch``` block catches all kinds of exceptions and throws again with wrapping QueryInterruptedException (https://github.com/druid-io/druid/pull/3939/files/4f36f619bdfe5f7084913e77c583cf2f423d304a#diff-852ac93b1541cb9178ad922dc30be4baR176). This line causes QueryInterruptedException is wrapped twice unnecessarily, so I changed.